cassandra/src/main/scala/org/apache/zeppelin/cassandra/DisplaySystem.scala

```diff
@@ -507,11 +558,15 @@ object MetaDataConverter {
         .map(c => new ColumnDetails(c.columnMeta.getName, StaticColumn, c.columnMeta.getType))
     }
 
-    protected def convertClusteringColumns(columns: List[ColumnMetaWrapper], orders: List[DriverClusteringOrder]): List[ColumnDetails] = {
+    protected def convertClusteringColumns(
+      columns: List[ColumnMetaWrapper],
+      orders: List[DriverClusteringOrder]
+    ): List[ColumnDetails] = {
```

Comment:
@jongyoul Do you think which style we should choose? Show a sample please, I'll re-check all the scala code.